app/components/common/navbar/index.tsx

```diff
@@ -108,6 +110,7 @@ const Navbar = () => {
                 classShortHands.flexDesktopRow
               } items-center mb-0 md:mb-8 cursor-pointer group`}
               onClick={() => toggle(navIntem.id)}
+              key={id++}
```

Comment:
`navbarPages` already has an `id`, you don’t have to declare and use it separately.